intercom_flutter/ios/Classes/IntercomFlutterPlugin.m

```diff
@@ -188,4 +190,40 @@ - (ICMUserAttributes *) getAttributes:(FlutterMethodCall *)call {
     
     return attributes;
 }
+
+- (NSData *) createDataWithHexString:(NSString*)inputString {
```

Comment:
As I understand, this function takes a string and returns that string in hexadecimal format, right? Is there any language function/feature that does that?

Reply:
I don't know. But As I checked the other PR that is also doing the same thing https://github.com/v3rm0n/intercom_flutter/pull/79/files#:~:text=byte_chars-,%5B,-0%5D%20%3D%20%5Bstring%20characterAtIndex%3Ai